scout/load/panelapp.py

```diff
@@ -28,7 +28,7 @@ def load_panelapp_panel(
         LOG.info("Fetching all panel app panels")
         panel_ids: List[str] = panelapp.get_panel_ids(signed_off=False)
 
-    for _ in panel_ids:
```

Comment:
I think this might have been broken since we switched to the new API, since before we didn't need to collect the single panels IDs, apparently!